frontend/src/js/tooltip/Tooltip.tsx

```diff
@@ -177,6 +198,17 @@ const Tooltip = () => {
               />
             )}
           </PinnedLabel>
+          {parent !== label && parent ? (
+            <IndentRoot>
```

Comment:
This looks pretty squished right underneath the parent label. Could you give this some space to top and bottom? Maybe something like `margin: 3px 0 8px`?